datastore/datastore.go

```diff
@@ -679,3 +701,16 @@ func (c *Client) Mutate(ctx context.Context, muts ...*Mutation) (ret []*Key, err
 	}
 	return ret, nil
 }
+
+// ReadOptions provides specific instructions for how to access documents in the database.
+// Currently, only ReadTime is supported.
+type ReadOptions struct {
```

Comment:
Without any context on what the future might hold, I might add a `WithReadTime` below and otherwise not export this struct. In the future you could add a `WithReadOptions` perhaps. In either case, I'm inclined to expand the API only as much as is needed and then leave the door open for a more generic API in the future. WDYT?

Reply:
Done.